File: backend/src/main/java/com/lawencon/elearn/dto/learningschedule/LearningScheduleInsertReqDto.java
package com.lawencon.elearn.dto.learningschedule;

public class LearningScheduleInsertReqDto {
	private String startTime;
	private String endTime;
	private Long learningId;
	public String getStartTime() {
		return startTime;
	}
	public void setStartTime(String startTime) {
		this.startTime = startTime;
	}
	public String getEndTime() {
		return endTime;
	}
	public void setEndTime(String endTime) {
		this.endTime = endTime;
	}
	public Long getLearningId() {
		return learningId;
	}
	public void setLearningId(Long learningId) {
		this.learningId = learningId;
	}
}

File: frontend/src/app/dto/learning-schedule/learning-schedule.insert.req.dto.ts
export class LearningScheduleInsertReqDto {

    private startTime! : string
	private endTime! : string
	private learningId! : number

    public get _startTime(): string {
        return this.startTime;
    }

    public set _startTime(_startTime: string) {
        this.startTime = _startTime;
    }

    public get _endTime(): string {
        return this.endTime;
    }

    public set _endTime(_endTime: string) {
        this.endTime = _endTime;
    }

    public get _learningId(): number {
        return this.learningId;
    }

    public set _learningId(_learningId: number) {
        this.learningId = _learningId;
    }

}